pom.xml

```diff
@@ -199,7 +199,7 @@
                     <plugin>
                         <groupId>com.github.spotbugs</groupId>
                         <artifactId>spotbugs-maven-plugin</artifactId>
-                        <version>4.5.3</version>
+                        <version>4.8.3</version>

```

Comment:
Not sure how this works, but there is no such version in central.

Reply:
You're right, thanks for spotting this!
4.5.3 does not exist either, but the latest is 4.8.3.1